.github/workflows/nightly_tests.yml

```diff
@@ -42,3 +42,65 @@ jobs:
     - name: Cleaning
       if: ${{ always() }}
       run: ./scripts/clean_fcctr.sh
+
+  cri-tests:
+    name: CRI tests (nightly)
+    runs-on: [self-hosted, cri]
+    strategy:
+      fail-fast: false
+      matrix:
+        vhive_args: ["-dbg", "-dbg -snapshots", "-dbg -snapshots -upf"]
+
+    steps:
+
+    - name: Host Info
+      env:
+          GITHUB_RUN_ID: ${{ github.run_id }}
+          GITHUB_RUNNER_VHIVE_ARGS: $${{ matrix.vhive_args }}
+      run: |
+        echo $HOSTNAME
+        echo $GITHUB_RUN_ID
+
+    - name: Setup TMPDIR
+      run: mkdir -p $HOME/tmp
+
+    - name: Set up Go 1.15
+      env:
+          GOROOT: $HOME/go
+          GOCACHE: /root/tmp/gocache
+          GOPATH: /root/tmp/gopath
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Setup firecracker-containerd
+      run: ./scripts/setup_firecracker_containerd.sh
+
+    - name: Build
+      env:
+          GOCACHE: /root/tmp/gocache
+          GOPATH: /root/tmp/gopath
+      run: go build
```

Comment:
Please remove this since we are executing `go build` inside the `start_onenode_vhive_cluster.sh` script.

Reply:
I think that it's better to have an explicit step here. the build inside the script is for interactive usage